src/dat.c

```diff
@@ -1,5 +1,14 @@
 #include "erfa.h"
 
+static eraLEAPSECOND *changes = 0;
+static int NDAT = 0;
```

Comment:
`changes` and `NDAT` are moved outside the scope of the `eraDat()` function, in order to be accessible by `eraUpdateLeapSeconds`.